### DISCUSSION TITLE
Add support for opacity (alpha) on color field type

### DISCUSSION THREAD

**Discussion by sc0ttkclark** [pods-framework/pods#6808] [Ideas]
A few changes to make this possible:

* In our Block implementation, we should support removing `disableAlpha`
* In our DFV field type for color, we should support removing `disableAlpha`
* Add the UI option to the color field type

The caveat here is that Iris does not support alpha so this would only be supported for DFV/Block fields and not any fields using `'disable_dfv' => true`

**Comment by vesna-droid** [comment]
Could color field also have an option of accepting color names? Like red, blue, cyan.., etc. Could be useful for our users who have names for their official colors but don't know the hex code of it.

**Comment by JoryHogeveen** [comment]
Hi @vesna-droid 
This would be unrelated to this topic. Please create a new one.
Keep in mind that we are using WordPress core features here, as long as WP core doesn't support this we probably won't either.

One thing we might include at some point are predefined color pallets from your theme.
